solr/conf/schema.xml

```diff
@@ -143,10 +143,8 @@
     <fieldType name="location" class="solr.LatLonType" subFieldSuffix="_d"/>
 
     <fieldType name="location_rpt" class="solr.SpatialRecursivePrefixTreeFieldType"
-               distErrPct="0.025"
-               maxDistErr="0.000009"
-               units="degrees"
-            />
+               geo="true" distErrPct="0.025" maxDistErr="0.001" distanceUnits="kilometers"/>
+
```

Comment:
do we use distance filtering in GeoBlacklight? the change in units here might be problematic if we do.